New Player Guide

Turtlelord26 opened the pull request:
From Issue #207. Removed favored class text and added some of a character creation walkthrough aimed at new players.

Looking for a go-ahead from both technical and creative because I am trying to accomplish a creative goal (new-player-friendly) with a technical document.

Turtlelord26 committed:
Favored Class Text Deleted

Also aligned Opaleite text to match the rest of the file.

Turtlelord26 committed:
Add Pointer to New Player Walkthrough

Turtlelord26 committed:
Create 00_New_Player_Walkthrough.txt

Still WIP, waiting on scheduled fundamental game updates.

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

Markdown is a thing that requires specific syntax.

bleehu commented on CharacterCreation/01_Races.txt:
```diff
@@ -158,11 +154,31 @@ have this augment installed.
+
+
+Trioch II is now a place for venturing archaeologists to come and investigate the 
+way the Opaleites used to live. 
```
Thank you for justifying this! +1

bleehu commented on CharacterCreation/00_New_Player_Walkthrough.txt:
```diff
@@ -0,0 +1,47 @@
+*Section to be added on account of impending addition of this feature*
+
+## Step 4: Attribute Point Allocation
+Now that you know what your character is capable of in-combat (your class) and out-of-combat (your non-combat skills), 
```
I know it's an intentional and helpful oversimplification, but I don't like saying that your class is for in-combat and you skills are for out-of-combat.

Turtlelord26 replied:
Found a way to restate.

bleehu commented on CharacterCreation/00_New_Player_Walkthrough.txt:
```diff
@@ -0,0 +1,47 @@
+2. Have at least 6 in either STR or DEX, and if possible up to 8. One of these stats will determine your movement speed,
+which is of high value.
+3. Have at least 6 Luck. Luck is a unique attribute and its value can be hard to gauge without experiencing its effects 
+firsthand.
```
I would suggest having at least 4 luck. I wouldn't say that your first character would need to be 'lucky'.

Turtlelord26 replied:
Fair enough. I can settle for just advising them to not be actively unlucky.

bleehu replied:
super nit-pick, the space is missing between "attribute pointes_to allocoate"

bleehu reviewed (changes requested):
Most of these changes are non-blocking nitpicks, but before we merge, I would like to insist that we not merge with dev notes of future work. Either we write up how we want Races/Classes/Backgrounds to work and hold merging this from dev to RC, or preferably we yank the notes and add them as the features are ready.

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

Addressing bleehu review.

Turtlelord26 committed:
Space Created

I can hardly complain when you nitpick me given how much I nitpick everyone else.

bleehu reviewed (approved):
Make Shipit! Well done, Turtle!

Turtlelord26 commented:
@Reveraine This seems like a space you should live in so I would like your go-ahead too.
In particular, someone less technical should look at the document for clarity.

Reveraine commented on CharacterCreation/00_New_Player_Walkthrough.txt:
```diff
@@ -31,17 +31,17 @@ on what the attributes themselves do and the dependencies of the skills you sele
 
-##Step 7: Equipment
+## Step 7: Equipment
 Any remaining moneys you have can be spent on miscellaneous equipment from the [Items](https://github.com/bleehu/Compound_X/blob/play/CharacterCreation/07_Items.txt) document. Take whatever items you
```
This should probably say 'credits' instead of money, but that's not a big change.

bleehu replied:
Good Aesthetic Catch!

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

credits > money
